test/choice-rule-with-comment.json

```diff
@@ -0,0 +1,21 @@
+{
+    "StartAt": "p",
+    "States": {
+      "p": {
+        "Type": "Choice",
+        "Choices": [
+      {
+        "Variable": "$.foo",
+        "StringEquals": "x",
+        "Next": "x",
+        "Comment": "foo"
+      }
+        ],
+        "ResultPath": "$.foo"
```

Comment:
this won't succeed validation - did you maybe mean OutputPath?

Reply:
that'll teach me for copying and pasting